File: JS/Car.js
/*function Car(licence, driver) {
    this.id;
    this.license = licence;
    this.driver = driver;
    this.passenger;

    Car.prototype.printDataCar = function () {
        console.log(this.driver)
        console.log(this.driver.name)
        console.log(this.driver.document)
    }
}*/
//------------new method------------
class Car{
    #passenger
    constructor(license, driver, passenger){
        this.id;
        this.license = license;
        this.driver = driver;
        this.#passenger = passenger;
    }
    
    get passenger(){
        return this.#passenger
    }
    set passenger(pasajeros){
        if(pasajeros < 4){
            console.error('se requiere una cantidad mayor de pasajeros');
        }else{
            
            this.#passenger = pasajeros;
        }

    }

    printDataCar() {
        console.table(this.driver);
        /*console.log(this.driver.id);
        console.log(this.driver.name);
        console.log(this.driver.document);
        console.log(this.driver.email);*/
        console.log(this.passenger);
    }
}

File: Python/car.py
from account import Account


class Car:
    id = int
    licence = str
    driver = Account('', '', '')
    __passenger = int
    
    #constructor
    def __init__(self, licence, driver):
        #self.id
        self.licence = licence
        self.driver = driver

    #metodo getter
    @property
    def passenger(self):
        return self.__passenger

    #metodo setter
    @passenger.setter
    def passenger(self, passengers: int):
        if passengers == 4:
            self.__passenger = passengers
        else:
            print("necesitas tener una cantidad mayor de pasajeros")
